### PR TITLE
feat: display the latest 8 metaphor stories in home page

### PR DESCRIPTION
**Pull request by darkterminal**
## 📓 What Your Metaphor

The homepage now **_displays the latest 8 posts_** from the `public/collections/stories` folder.

## 📷 What Your Punk Signature?

image

image

Screen Shot 2023-04-16 at 10 53 50

### CODE REVIEW

**Review by mkubdev — APPROVED** [review]
L G T M. ⚡🔥

File: src/components/Footer.js [hunk 1]
```diff
@@ -5,30 +5,30 @@ import { GiSheep } from 'react-icons/gi';
 function Footer() {
 	return (
 		<div className="mt-auto">
-			<footer className="absolute bottom-0 w-full dark:bg-white bg-zinc-900 px-4 py-2 border-t-2 sm:flex sm:items-center sm:justify-between">
+			<footer className="absolute bottom-0 w-full px-4 py-2 border-t-2 sm:flex sm:items-center sm:justify-between backdrop-filter backdrop-blur-lg backdrop-opacity-90">
```
[the remark this answers is not in the document]
Comment (darkterminal) [reply src/components/Footer.js:8]: Did I need to refactoring it?

[the remark this answers is not in the document]
Comment (mkubdev) [reply src/components/Footer.js:8]: No just sharing tip you maybe already know. I close!

[the remark this answers is not in the document]
Comment (darkterminal) [reply src/components/Footer.js:8]: Wow... thank you :smiley:

**Pull request merged**